episodes/4-advanced-layer-types.Rmd

```diff
@@ -411,16 +426,21 @@ outputs = keras.layers.Dense(10)(x)
 ## Solution
 We add an extra Conv2D layer after the second pooling layer:
 ```python
-inputs = keras.Input(shape=train_images.shape[1:])
-x = keras.layers.Conv2D(50, (3, 3), activation='relu')(inputs)
-x = keras.layers.MaxPooling2D((2, 2))(x)
-x = keras.layers.Conv2D(50, (3, 3), activation='relu')(x)
-x = keras.layers.MaxPooling2D((2, 2))(x)
-x = keras.layers.Conv2D(50, (3, 3), activation='relu')(x)
-x = keras.layers.Flatten()(x)
-x = keras.layers.Dense(50, activation='relu')(x)
-outputs = keras.layers.Dense(10)(x)
-model = keras.Model(inputs=inputs, outputs=outputs, name="cifar_model")
+def create_nn():
```

Comment:
Would it be a better idea to give this function a different name? maybe create_nn_with_extra_layer()?

Reply:
Totally agree! Changed it